packages/styleguide/lib/styleguide-view.js

```diff
@@ -169,86 +169,158 @@ export default class StyleguideView {
           </StyleguideSection>
 
           <StyleguideSection onDidInitialize={this.didInitializeSection.bind(this)} name='icons' title='Icons'>
-            <p>Atom comes bundled with the Octicons. It lets you easily add icons to your packages.</p>
-            <p>Currently version <code>4.4.0</code> is available. In addition some older icons from version <code>2.1.2</code> are still kept for backwards compatibility. Make sure to use the <code>icon icon-</code> prefix in front of an icon name. See the <a href='http://flight-manual.atom.io/hacking-atom/sections/iconography/'>documentation</a> for more details.</p>
+            <p>Pulsar comes bundled with the Octicons. It lets you easily add icons to your packages.</p>
```

Comment:
```suggestion
            <p>Pulsar comes bundled with <a href='https://primer.style/design/foundations/icons/'>Octicons</a>. It lets you easily add icons to your packages.</p>
```

I know the `the` comes from the original but I find it really weird. Any objections to removing it? The primer/octicons repo doesn't seem to have it in this style either.

I've also suggested a link to `Octicons` to the Octicons website as I think this is just generally helpful.